### PR TITLE
Landing Page

### PR DESCRIPTION
**Pull request by isaac342368**

### CODE REVIEW

File: Components/LandingPage.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import { View, Button, StyleSheet, Text, TouchableOpacity } from 'react-native';
+import { LinearGradient } from 'expo-linear-gradient';
+import React from 'react';
+import { Video } from 'expo-av';
+
+const LandingPage = ({ navigation }) => {
+  return (
+    <LinearGradient
+      colors={['#6a11cb', '#2575fc']}
```
Comment (jlipscomb071) [inline Components/LandingPage.jsx:9]: I think the landing page is pretty blue. I'd suggest a predominantly white landing page with blue accents. The blue makes certain text harder to read.

File: Components/LandingPage.jsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      <Video
+        source={require('./Homie.mp4')}
+        rate={1.0}
+        isMuted={true}
+        resizeMode="contain"
+        shouldPlay
+        isLooping
+        style={styles.logoVideo}
+      />
+      <Text style={styles.welcomeText}>Welcome to HomieHub</Text>
```
Comment (jlipscomb071) [inline Components/LandingPage.jsx:23]: The font of the Welcome text could be a bit more unique. I know we didn't agree on a font, so maybe we can agree on one moving forward.

File: Components/LandingPage.jsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+      <Text style={styles.descriptionText}>
+        HomieHub is the ultimate tool to keep you and your roommates organized and connected.
+      </Text>
+      
+      <Text style={styles.memberText}>Already a member?</Text>
+      <TouchableOpacity style={styles.loginButton} onPress={() => navigation.navigate('Login')}>
+        <Text style={styles.buttonText}>Log In</Text>
```
Comment (jlipscomb071) [inline Components/LandingPage.jsx:30]: I think rounded buttons would make the page more modern.

File: Components/LandingPage.jsx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+      </TouchableOpacity>
+      <TouchableOpacity style={styles.registerButton} onPress={() => navigation.navigate('Register')}>
+        <Text style={styles.buttonText}>Register</Text>
+      </TouchableOpacity>
+    </LinearGradient>
+  );
+};
+
+const styles = StyleSheet.create({
```
Comment (jlipscomb071) [inline Components/LandingPage.jsx:39]: Maybe we should store this css in a separate file? I'd assume we'll end up recycling some of it for other pages.

**Review by jlipscomb071 — COMMENTED** [review]
Left some comments. We should meet to talk a bit more about the design of the landing page.

File: Components/LandingPage.jsx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    padding: 15,
+    borderRadius: 5,
+    width: '100%', // Full width
+    marginBottom: 10,
+  },
+  registerButton: {
+    backgroundColor: '#4a09a5', // Darker shade of blue
+    padding: 15,
+    borderRadius: 5,
+    width: '100%', // Full width
```
Comment (aarontorres0) [inline Components/LandingPage.jsx:75]: nit: comment not necessary

File: Components/LandingPage.jsx [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+        HomieHub is the ultimate tool to keep you and your roommates organized and connected.
+      </Text>
+      
+      <Text style={styles.memberText}>Already a member?</Text>
+      <TouchableOpacity style={styles.loginButton} onPress={() => navigation.navigate('Login')}>
+        <Text style={styles.buttonText}>Log In</Text>
+      </TouchableOpacity>
+      <TouchableOpacity style={styles.registerButton} onPress={() => navigation.navigate('Register')}>
+        <Text style={styles.buttonText}>Register</Text>
+      </TouchableOpacity>
```
Comment (aarontorres0) [inline Components/LandingPage.jsx:34]: I've noticed most apps have the register button and then the login button. We could switch the order of these buttons here

File: Components/LandingPage.jsx (hunk 1 above)
Comment (aarontorres0) [inline Components/LandingPage.jsx:9]: These colors are pretty dark and similar. Having the background be white would provide a better contrast

File: Components/LandingPage.jsx [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    padding: 20,
+  },
+  logoVideo: {
+    width: '100%',
+    height: '60%', // Made video larger by occupying 50% of screen height
```
Comment (aarontorres0) [inline Components/LandingPage.jsx:46]: nit: unnecessary comment

**Review by aarontorres0 — CHANGES_REQUESTED** [review]

**Comment by jlipscomb071** [comment]
We're gonna merge the other PR instead.

**Pull request closed without merging**